### DISCUSSION TITLE
Howtos (e-Rechnung & Sales)?

### DISCUSSION THREAD

**Discussion by JPRuehmann** [tine-groupware/tine#85] [General]
Anyone knows a Step by Step Howto for the using of the Sales Module?
Especialy for viewing and creating e-Rechnungen?
Thanks.

**Comment by pschuele** [comment]
hi,
we'll let you know when we have a howto / video / etc. for the sales app and the new e-invoice feature.

**Reply by JPRuehmann** [reply]
Thanks / Danke.
